qa/tasks/cephfs/test_volumes.py

```diff
@@ -302,6 +305,30 @@ def test_volume_rm(self):
         else:
             raise RuntimeError("expected the 'fs volume rm' command to fail.")
 
+    def test_volume_rm_when_mon_delete_pool_false(self):
+        """
+        That the volume can only be removed when --yes-i-really-mean-it is used
+        and verify that the deleted volume is not listed anymore.
+        """
+        try:
+            self._raw_cmd("config", "set", "mon", "mon_allow_pool_delete", "false")
+            self._fs_cmd("volume", "rm", self.volname, "--yes-i-really-mean-it")
+        except CommandFailedError as ce:
+            if ce.exitstatus != errno.EPERM:
+                raise RuntimeError("expected the 'fs volume rm' command to fail with EPERM, "
+                                   "but it failed with {0}".format(ce.exitstatus))
+            else:
+                self._raw_cmd("config", "set", "mon", "mon_allow_pool_delete", "true")
+                self._fs_cmd("volume", "rm", self.volname, "--yes-i-really-mean-it")
+
```

Comment:
Check for `ENOENT` here ?

Reply:
Why should we do this?

Reply:
That's unnecessary. Just check if the volume and the pools are gone. That's it.